Lib/urlparse.py

```diff
@@ -180,8 +180,9 @@ def _checknetloc(netloc):
         return
     for c in '/?#@:':
         if c in netloc2:
-            raise ValueError(u"netloc '" + netloc + u"' contains invalid " +
-                             u"characters under NFKC normalization")
+            raise ValueError("netloc %r contains invalid characters "
+                             "under NFKC normalization"
+                             % netloc2)
```

Comment:
Should `netloc` be reported in the error message? Python 3 and previous error messages report `netloc`.

Reply:
Oh. You're right. I also fixed this bug and I updated the unit test to test this case.